tests/event_store/conftest.py

```diff
@@ -25,3 +29,17 @@ def event_store_factory(request: SubRequest) -> BackendFactory:
     mark.xfail_if_not_implemented_yet(request, backend_name)
     mark.skip_backend(request, backend_name)
     return cast(BackendFactory, request.getfixturevalue(backend_name))
+
+
+@pytest.fixture
+def other_client(event_store_factory: BackendFactory) -> Iterator[OtherClient]:
```

Comment:
Subscriptions are complex and lets move subscription tests to separate subpackage.

Reply:
Done